packages/mc-html-template/lib/load-headers.js

```diff
@@ -83,6 +83,8 @@ module.exports = (env, options) => {
       ),
       'connect-src': [
         "'self'",
+        'mc-api.europe-west1.gcp.commercetools.com',
+        'mc-api.us-central1.gcp.commercetools.com',
```

Comment:
This allows a local development environment to work with them by default without changing cors rules. We can then in a loooong time (😉) remove the old urls at some point.